rules/opentitan_gdb_test.bzl

```diff
@@ -0,0 +1,122 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@lowrisc_opentitan//rules:rv.bzl", "rv_rule")
+
+def _opentitan_gdb_fpga_cw310_test(ctx):
+    test_script = '''#!/usr/bin/env bash
+        set -e
+
+        COLOR_RED='\\x1b[0;31m'
+        COLOR_GREEN='\\x1b[0;32m'
+        COLOR_PURPLE='\\x1b[0;35m'
+
+        function prefix_lines() {{
+            LABEL="$1"
+            COLOR="$2"
+            COLOR_RESET='\\x1b[m'
+            sed -Eu "s/(.*)/$COLOR[$LABEL]$COLOR_RESET \\1/"
+        }}
+
+        set -x
+
+        ./opentitantool load-bitstream --rom-kind=TestRom rom.bit
+
+        (openocd -f /usr/share/openocd/scripts/interface/ftdi/olimex-arm-usb-tiny-h.cfg \\
+            -c "adapter speed 500; transport select jtag; reset_config trst_and_srst" \\
+            -f {openocd_earlgrey_config} 2>&1 \\
+            | prefix_lines OPENOCD "$COLOR_PURPLE") &
+        OPENOCD_PID=$!
+
+        # For debugging, it may be useful to use `--init-command`, which causes
+        # GDB to drop to the interactive prompt when the script ends rather than
+        # exiting.
+        (/tools/riscv/bin/riscv32-unknown-elf-gdb --command=script.gdb 2>&1 \\
+            | prefix_lines GDB "$COLOR_GREEN") &
+
+        ./opentitantool console \\
+            --timeout 15s \\
+            --exit-success='{exit_success_pattern}' \\
+            | prefix_lines CONSOLE "$COLOR_RED"
```

Comment:
```suggestion
            --exit-success='{exit_success_pattern}' 2>&1 \\
            | prefix_lines CONSOLE "$COLOR_RED"
```

Reply:
Done. I learned from `man bash` that `foo |& bar` is shorthand for `foo 2>&1 | bar`, so I'm just using that syntax.